scania-digital-design-system/sdds
# feat:(radio-button) migrate radio button

**Describe pull-request**  
Migrate radio button component

**Solving issue**  
Fixes: AB#2414

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below
2. Check in component -> radio button
3. Check that the design is aligned with figma. 

**Screenshots**  
-

**Additional context**  
-
